src/cli.ts

```diff
@@ -4,14 +4,23 @@ import { type BetterAuthOptions, logger } from "better-auth"
 import { loadConfig } from "c12"
 import { defineCommand, runMain } from "citty"
 import { version } from "../package.json"
-import { generateRemultSchema } from "./remult-generate-schema"
-import { RemultBetterAuthError } from "./utils"
```

Comment:
this cli will be gone soon any way. I prefer not to maintain it and use better-auth/cli